lms/djangoapps/grades/signals/signals.py

```diff
@@ -78,3 +78,8 @@
         'subsection_grade',  # SubsectionGrade object
     ]
 )
+
+
+# Signal that indicates that a user's grade for a course has been updated.
+# This is a downstream signal of SUBSECTION_SCORE_CHANGED.
+from openedx.core.djangoapps.signals.signals import COURSE_GRADE_CHANGED
```

Comment:
Why was this added here? Can it go at the top of the file near the other import statement, or is this location deliberate?